crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -309,6 +352,10 @@ pub trait BuildChildren {
     /// will have those children removed from its list. Removing all children from a parent causes its
     /// [`Children`] component to be removed from the entity.
     fn add_child(&mut self, child: Entity) -> &mut Self;
+    /// Clear children
+    fn clear_children(&mut self) -> &mut Self;
+    /// Replace children
```

Comment:
```suggestion
    /// Removes all current children from this entity, replacing them with the specified list of entities.
    /// This will add a [`Children`] component to this entity if one does not already exist.
```